string gtimu
int32 imu_gps_week
float64 imu_gps_time
float64 imu_gyro_x
float64 imu_gyro_y
float64 imu_gyro_z
float64 imu_acc_x
float64 imu_acc_y
float64 imu_acc_z
string gpfpd
int32 fpd_gps_week
float64 fpd_gps_time
float64 fpd_heading
float64 fpd_pitch
float64 fpd_roll
float64 fpd_lattitude
float64 fpd_longitude
float64 fpd_altitude